index.js

```diff
@@ -88,8 +88,14 @@ exports.stringifyRequest = function(loaderContext, request) {
 	var splitted = request.split("!");
 	var context = loaderContext.context || (loaderContext.options && loaderContext.options.context);
 	return JSON.stringify(splitted.map(function(part) {
-		if(/^\/|^[A-Z]:/i.test(part) && context)
-			return "./" + path.relative(context, part).replace(/\\/g, "/");
+		if(/^\/|^[A-Z]:/i.test(part) && context) {
+			part = path.relative(context, part).replace(/\\/g, "/");
```

Comment:
Could you move the `replace` to the else case? Absolute windows path should have `\`.